spec/spec_helper.rb

```diff
@@ -6,3 +6,42 @@
   Coveralls::SimpleCov::Formatter
 ]
 Coveralls.wear!
+
+ENV['RACK_ENV'] = 'test'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.